components/i18n/src/test/java/org/datacleaner/beans/CharacterSetDistributionAnalyzerTest.java

```diff
@@ -130,4 +150,76 @@ public void testSimpleScenario() throws Exception {
         assertEquals("Thaana                0      0 ", resultLines[23]);
         assertEquals("Thai                  0      0 ", resultLines[24]);
     }
+
+    public void testOutputDataStream() throws Throwable {
+        final MultiThreadedTaskRunner taskRunner = new MultiThreadedTaskRunner(16);
+        final DataCleanerEnvironment environment = new DataCleanerEnvironmentImpl()
+                .withTaskRunner(taskRunner);
+        final FileResource file = new FileResource("src/test/resources/strings.txt");
```

Comment:
I would prefer to have a more informative name for the file. Maybe testOutputDataStream.txt?